### PR TITLE
Remove some short options

### PR DESCRIPTION
**Pull request by dysfunctor**
- [x] Set a valid title
- [x] Use branch-name `[<ref>-]<feature-description>`
- [x] Brief description of the changes
- [x] Create matching PRs in dependent repos

Removed most of the short version of options. I've kept -n for number of proofs and -w for wait, as they could be commonly used.

### CODE REVIEW

File: upa/src/tool/submitProofs.ts [hunk 1]
```diff
@@ -55,7 +54,6 @@ export const submitProofs = command({
     isDryRun: flag({
       type: boolean,
       long: "dry-run",
-      short: "n",
```
Comment (dtebbs) [inline upa/src/tool/submitProofs.ts]: This one is quite standard.  I'd be keep to keep it and remove or update the one above.

File: upa/src/tool/options.ts [hunk 2]
```diff
@@ -35,7 +35,6 @@ export function keyfile(
   return option({
     type: string,
     long: "keyfile",
-    short: "k",
```
Comment (dtebbs) [inline upa/src/tool/options.ts]: Keep?

File: upa/src/tool/options.ts [hunk 3]
```diff
@@ -82,7 +80,6 @@ export function endpoint(): Option {
   return option({
     type: string,
     long: "endpoint",
-    short: "e",
```
Comment (dtebbs) [inline upa/src/tool/options.ts]: Keep?

**Review by dtebbs — APPROVED** [review]
Couple I'd like to keep, otherwise no objection.

File: upa/src/tool/submitProofs.ts (hunk 1 above)
Comment (dysfunctor) [reply upa/src/tool/submitProofs.ts]: Ok, removed the `numProofs` one

File: upa/src/tool/options.ts (hunk 2 above)
Comment (dysfunctor) [reply upa/src/tool/options.ts]: put back in

File: upa/src/tool/options.ts (hunk 3 above)
Comment (dysfunctor) [reply upa/src/tool/options.ts]: put back in

**Pull request merged**